src/objects/ModmailConversationAuthor.d.ts

```diff
@@ -0,0 +1,40 @@
+import Subreddit from './Subreddit';
+import RedditContent from './RedditContent';
+import RedditUser from './RedditUser';
+
+export interface BanStatus {
+    endDate?: string | null;
+    reason: string;
+    isBanned: boolean;
+    isPermanent: boolean;
+}
+
+// TODO: I couldn't find any definitions for this, nor
+// get the field to appear in testing
+export type RecentPost = any;
```

Comment:
Recent posts appear if they have had posts in the subreddit the modmail is part of. It returns:
```json
{
  "recentPosts": {
    "t3_ckem48": {
      "date": "2019-07-31T21:17:34.209000+00:00",
      "permalink": "https://www.reddit.com/r/EliteDangerous/comments/ckem48/dear_fdev_please_have_mercy_and_kill_this/",
      "title": "Dear FDev, Please Have Mercy And Kill This Gameplay (please read the first comment)"
    },
    "t3_cgf36r": {
      "date": "2019-07-22T16:03:46.261000+00:00",
      "permalink": "https://www.reddit.com/r/EliteDangerous/comments/cgf36r/quick_visit_to_the_local_guardian_tech/",
      "title": "Quick Visit To The Local Guardian Tech Distribution Centre"
    },
    "t3_cgy0al": {
      "date": "2019-07-23T20:29:20.295000+00:00",
      "permalink": "https://www.reddit.com/r/EliteDangerous/comments/cgy0al/atmospheric_planetary_landing_coming_december_13/",
      "title": "Atmospheric Planetary Landing Coming December 13, 2019"
    }
  }
}
```